### PR TITLE
Picker - add support for layout modifiers (margin, paddings)

### PR DESCRIPTION
**Pull request by Inbal-Tish**

### CODE REVIEW

**Comment by ethanshar** [comment]
@Inbal-Tish 
There's some conflicts, probably from merging the previous Picker PR

**Comment by Inbal-Tish** [comment]
Done.

File: src/components/picker/index.js [hunk 1]
```diff
@@ -385,10 +386,12 @@ class Picker extends PureComponent {
 
     const textInputProps = TextField.extractOwnProps(this.props);
     const label = this.getLabelValueText();
-    
+    const layoutProps = {...extractMarginValues(this.props), ...extractPaddingValues(this.props)};
```
Comment (ethanshar) [inline src/components/picker/index.js:389]: no need for that.. 
by wrapping a component with asBaseComponent you get injected with the `modifiers` prop. 
From there you can get whatever modifiers values you need (margins, paddings, colors, etc..) 

I would see how `View` is implemented for a reference.

**Review by ethanshar — CHANGES_REQUESTED** [review]

File: src/components/picker/index.js [hunk 2]
```diff
@@ -385,10 +392,18 @@ class Picker extends PureComponent {
...
+
     return (
       <TextField
         {...textInputProps}
+        containerStyle={[
+          containerStyle,
+          paddings,
+          margins,
+          alignments,
+          positionStyle
```
Comment (ethanshar) [inline src/components/picker/index.js:405]: `containerStyle` should be prioritized over modifiers style. 
`alignments` style is redundant in this case, it does't affect the component so there's no need to support it.

**Review by ethanshar — CHANGES_REQUESTED** [review]

File: src/components/picker/index.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/components/picker/index.js]: Done.

File: src/components/picker/index.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/picker/index.js]: Done.

**Review by ethanshar — APPROVED** [review]

**Comment by ethanshar** [comment]
@Inbal-Tish 
I approved this, but I see now build has failed. 
Can you check? 
You can merge once build is fixed..

**Pull request merged**